recipes-matter/barton-matter-reference/files/.gn

```diff
@@ -0,0 +1,42 @@
+# Copyright 2023 Comcast Cable Communications, LLC
```

Comment:
Incidentally, the Comcast string is: "Comcast Cable Communications Management, LLC".